litestar/contrib/prometheus/config.py

```diff
@@ -15,7 +15,7 @@
 try:
     import prometheus_client  # noqa: F401
 except ImportError as e:
-    raise MissingDependencyException("prometheus_client") from e
+    raise MissingDependencyException("prometheus_client", extra="prometheus") from e
```

Comment:
should this be `"prometheus-client"`?

https://pypi.org/project/prometheus-client/

Reply:
`"prometheus"` points to https://github.com/slok/prometheus-python which has been archived and no updates for 8 years.

Reply:
The `pip install` bit should use `prometheus_client` (which *should* be an alias to `prometheus-client` IIRC because pypi? might be worth having `install_package="prometheus-client"` though for clarity's sake.) -- `extra` is specifically just for the extra recommendation and doesn't change the package name being recommended.

Reply:
Oh man - I'm getting all tangled up on this. Agree, `"prometheus"` for the `extra=...` and `"prometheus-client"` for the `install_package=...`.